apps/web/src/pages/workflow/index.tsx

```diff
@@ -67,27 +71,28 @@ const Workflow = () => {
 
     // ---------- 数据删除相关逻辑 ----------
     const confirm = useConfirm();
+    const warnIcon = useMemo(() => {
+        return <ErrorIcon sx={{ fill: '#F57C00' }} />;
```

Comment:
Do not use color values directly, use color variables instead

Reply:
![image](https://github.com/user-attachments/assets/95c3d6b3-054f-4966-b299-ab1905c9aba5)